1-js/03-code-quality/03-comments/article.md

```diff
@@ -1,38 +1,38 @@
-# Comments
+# टिप्पणियाँ
 
-As we know from the chapter <info:structure>, comments can be single-line: starting with `//` and multiline: `/* ... */`.
```

Comment:
Please don't translate these  `<info:structure>`. As you can see here https://javascript.info/comments it's the link to another page.